### PR TITLE
Release/1.4.1

### PR DESCRIPTION
**Pull request by Phips0812**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [submodule "src/ondewo-s2t-api"]
 	path = src/ondewo-s2t-api
 	url = https://github.com/ondewo/ondewo-s2t-api
+	branch = release/1.4.0
```
Comment (gaborod16) [inline .gitmodules:4]: Oh!
I see what the problem was before!

Yes, tag are sadly not supported.

File: RELEASE.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
 # Release History
 *****************
 
+## Release ONDEWO-S2T Angular Client 1.4.1
+
+### New Features
+* Upgraded from ngx-grpc 0.3.1 to 2.1.0
```
Comment (gaborod16) [inline RELEASE.md:7]: :+1:

File: package.json [hunk 3]
```diff
@@ -1,19 +1,27 @@
 {
   "name": "@ondewo/s2t-client-angular",
-  "version": "1.4.0",
-  "description": "Ondewo speech to text client library for angular",
+  "version": "1.4.1",
+  "description": "Ondewo natural language understanding client library for angular",
```
Comment (gaborod16) [inline package.json:4]: Gotcha!
This is the Speech-to-Text client, not NLU :)

**Review by gaborod16 — CHANGES_REQUESTED** [review]
Just a string change in the package.json file.
The rest looks consistent to its nature and purpose :)

**Comment by Phips0812** [comment]
Requested changes implemented -> Merging

**Pull request merged**